main.js

```diff
@@ -73,6 +73,13 @@ function reminderTour(win){
 
 function firstInstallTour(win) {
   if (win.activeInstall || win.reminderTour) {
+    if (get('extensions.tabcentertest1@mozilla.com.tourComplete')) {
+      utils.sendPing('tour_completed_reminder_began', win);
```

Comment:
Let's change the sendPing method to take `details`, and specify the extra event data that way.